homeassistant/components/glances/__init__.py

```diff
@@ -27,6 +28,16 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
 
     await hass.config_entries.async_forward_entry_setups(config_entry, PLATFORMS)
 
+    if config_entry.data[CONF_VERSION] == 2:
+        async_create_issue(
```

Comment:
Can we maybe add a version of HA where we deprecate the support?

Reply:
I think half a year from now is good. If there's no new library version at that time we can prolong it.

Reply:
Thanks for checking this issue @MartinHjelmare !

There is already a new library version that breaks v2 support, merged in dev. See my comment below:
https://github.com/home-assistant/core/pull/105887#issuecomment-1869806333

Reply:
@fhoekstra , the updated library only issues a warning if v2 is used. 6 months should be enough to see users raise up any concerns about using glances 2.